koans/StringKoans.php

```diff
@@ -77,9 +77,9 @@ public function testStringsCanContinueOntoMultipleLines()
     public function testStringsCanBeWrappedInAHeredocSyntax()
     {
         $string = <<<EOT
```

Comment:
This EOT is wrong. Somebody left this were when he make a rebase

Reply:
By this do you mean to delete them?